basic/exercise5/script5.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+function firstSunday() {
+    for (var year = 2014; year <= 2050; year++) {
```

Comment:
Please do not hardode values 2014 and 2050 in the body of this function - use function parameters instead.
You may call this function providing 2014 and 2050 parameters.